packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -42,17 +71,50 @@ export const PanelChrome: React.FC<PanelChromeProps> = ({
   const headerStyles: CSSProperties = {
     height: headerHeight,
   };
-
+  const itemStyles: CSSProperties = {
+    minHeight: headerHeight,
+    minWidth: headerHeight,
+  };
   const containerStyles: CSSProperties = { width, height };
 
+  const handleMenuOpen = () => {};
+
+  const hasHeader = title || titleItems.length > 0 || menu;
+
   return (
     <div className={styles.container} style={containerStyles}>
-      <div className={styles.header} style={headerStyles}>
-        <div className={styles.headerTitle}>{title}</div>
-        {itemsRenderer(leftItems, (items) => {
-          return <div className={styles.leftItems}>{items}</div>;
-        })}
-      </div>
+      {hasHeader && !hoverHeader && (
+        <div className={styles.headerContainer} style={headerStyles} data-testid="header-container">
+          {title && <div className={styles.title}>{title}</div>}
```

Comment:
Missing `title` property on the div - think especially useful when the ellipsis kicks in.